core/src/main/scala/org/bitcoins/core/util/EnvUtil.scala

```diff
@@ -14,4 +14,9 @@ object EnvUtil {
   lazy val isCI: Boolean = Properties.envOrNone("CI").contains("true")
 
   def getVersion: String = getClass.getPackage.getImplementationVersion
+
+  def isNativeSecp256k1Disabled: Boolean = {
+    val secpDisabled = System.getenv("DISABLE_SECP256K1")
+    secpDisabled != null && (secpDisabled.toLowerCase == "true" || secpDisabled == "1")
```

Comment:
don't know what the context is for this but probably also need to check if it `isEnabled` because it could also be disabled due to binaries being missing

Reply:
This is just checking if the environment variable is set, not if the native libraries are available. This is useful for optimizations on CI machines when we want to reduce the number of test iterations because the flag is set.